### PR TITLE
Nested fix

### PR DESCRIPTION
**Pull request by chrismcv**
This seems to work, I'm not sure if its the perfect solution.... @paulmillr @yavorsky

### CODE REVIEW

File: dist/exim.js [hunk 1]
```diff
@@ -387,24 +387,21 @@ var Store = (function () {
...
-      var correctArgs = ["key", "value"].every(function (item) {
-        return typeof item === "string";
-      });
-      return correctArgs ? store[key] = value : false;
+      store[key] = value;
     };
 
     var getValue = function getValue(key) {
-      return key ? store[key] : Object.create(store);
+      return key ? store[key] : store;
```
Comment (yavorsky) [inline dist/exim.js]: I think we lose immutability there

Comment (yavorsky) [inline dist/exim.js]:
```
var smth = store.get('smth');
smth.a = 1;
store.get('smth').a // 1;
```

File: dist/exim.js [hunk 2]
```diff
@@ -387,24 +387,21 @@ var Store = (function () {
     var _this = this;
 
     var setValue = function setValue(key, value) {
-      var correctArgs = ["key", "value"].every(function (item) {
-        return typeof item === "string";
-      });
-      return correctArgs ? store[key] = value : false;
+      store[key] = value;
```
Comment (yavorsky) [inline dist/exim.js]: This is OK!

File: dist/exim.js (hunk 1 above)
Comment (paulmillr) [inline dist/exim.js]: 1. We can return cloned object
2. We can return sealed / frozen object

**Comment by paulmillr** [comment]
Sorry for delay. Just forgot about this.

@chrismcv is this optional if we don't use babel?

**Comment by chrismcv** [comment]
Not sure, Object.assign needs shimmed, not sure how browserify handles that

**Comment by paulmillr** [comment]
We'll need to replace `object.assign` with [the shim](https://github.com/sindresorhus/object-assign/blob/master/index.js).

**Pull request merged**